--- id3v2-0.1.12/Makefile	2010-03-26 23:28:34.000000000 +0000
+++ id3v2-0.1.12/Makefile	2010-05-30 01:33:39.000000000 +0000
@@ -2,19 +2,17 @@
 
 VERSION=0.1.12
 
-PREFIX=	/opt/local
-CXXFLAGS+=	-Wall -I${PREFIX}/include/ -DVERSION="\"${VERSION}\"" #-DSORT_RUNTIME
-LDFLAGS+=	-L${PREFIX}/lib/ 
+CXXFLAGS+=	-Wall -I$(PREFIX)/include/ -DVERSION="\"${VERSION}\"" #-DSORT_RUNTIME
 
 id3v2:	convert.o list.o id3v2.o genre.o
-	${CXX} ${LDFLAGS} -pedantic -Wall -g -o $@ $^ -lz -lid3
+	$(CXX) $(LDFLAGS) -pedantic -Wall -g -o $@ $^ -lz -lid3
 
 create_map: create_map.o
-	${CXX} -Wall -g -o $@ $^
+	$(CXX) -Wall -g -o $@ $^
 
 install: all
-	install -c -s id3v2 ${PREFIX}/bin/id3v2
-	nroff -man id3v2.1 > ${PREFIX}/share/man/man1/id3v2.1
+	install -c -s id3v2 $(DESTDIR)$(PREFIX)/bin/id3v2
+	nroff -man id3v2.1 > $(DESTDIR)$(PREFIX)/share/man/man1/id3v2.1
 
 clean:
 	rm -f id3v2 create_map core *.o
