--- id3v2-0.1.9/Makefile.old	2003-02-25 16:58:23.000000000 +0100
+++ id3v2-0.1.9/Makefile	2004-01-05 06:12:06.000000000 +0100
@@ -1,15 +1,14 @@
 all: id3v2
 
-PREFIX=	/usr/local
-CXXFLAGS+=	-I${PREFIX}/include/
-LDFLAGS+=	-L${PREFIX}/lib/
+PREFIX=	/usr/
+GPP=g++
 
 id3v2:	convert.o list.o id3v2.o
-	c++ ${LDFLAGS} -pedantic -Wall -lz -lid3 -g -o $@ id3v2.o list.o convert.o
+	$(GPP) $(LDFLAGS) -lid3 -o $@ id3v2.o list.o convert.o
 
 install: all
-	install -c -s id3v2 ${PREFIX}/bin/id3v2
-	nroff -man id3v2.1 > ${PREFIX}/man/man1/id3v2.1
+	install -c id3v2 $(DESTDIR)$(PREFIX)/bin/id3v2
+	nroff -man id3v2.1 > $(DESTDIR)$(PREFIX)/share/man/man1/id3v2.1
 
 clean:
 	rm -f id3v2 core *.o
